package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@inseefr/lunatic",
-	"version": "3.4.10",
+	"version": "3.4.11-rc.0",
```

Comment:
We don't like to bump in features PR, no? Makes the commit history a bit messy to know what each versions were about.